README.markdown

```diff
@@ -1,6 +1,6 @@
 _spray-json_ is a lightweight, clean and efficient [JSON] implementation in Scala.
 
-It sports the following features:
```

Comment:
Thanks, Brandon!
Is this really illegal English?
I think `sport` is used here as a transitive verb as described by http://www.merriam-webster.com/dictionary/sport (first transitive meaning).